docs/sources/concepts/community_components.md

```diff
@@ -0,0 +1,20 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/concepts/community_components/
+description: Learn about community components
+title: Community components
+weight: 100
+---
+
+# Community components
+
+__Community components__ are [components][Components] implemented and maintained by the community.
+
+While Grafana does not offer commercial support for these components, they undergo acceptance and review by {{< param "PRODUCT_NAME" >}}'s team before being added to the repository.
+
+To use these community components, you need to explicitly pass the `--community-component` flag to the `run` command.
+
+{{< admonition type="warning" >}}
+__Community components__ that are not kept up to date by the community may be disable or removed.
+{{< /admonition >}}
```

Comment:
I think this is a bit strongly worded. My understanding is that we're worried about community components lacking an owner and preventing us from upgrading our dependencies or forcing us to act as the owners. 

Given that, I think this is the criteria for removal:

* The community component has been abandoned by its current owner, and 
* We are unable to find a new owner to take over, and 
* The unowned component is preventing us from being able to continue work on Alloy

I think that last bullet point is fairly important; it makes it clearer that we only remove things when it gets in the way of other progress.